CanHazFunny/CanHazFunny/IJokeService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace CanHazFunny
+{
+    public interface IJokeService
+    {
+        public string GetJoke();
```

Comment:
For interfaces, avoid specifying access modifiers.
```suggestion
        string GetJoke();
```